topics/ratchet/index.md

```diff
@@ -0,0 +1,12 @@
+---
+related: php, websockets
+created_by: Chris Boden
+display_name: Ratchet
+logo: ratchet.png
+github_url: https://github.com/ratchetphp
+released: May 11, 2012
+short_description: Ratchet is a set of libraries to handle WebSockets asynchronously in PHP.
+topic: ratchet
+url: http://socketo.me/
+---
+Ratchet is a set of libraries that allows PHP developers to create real time, bi-directional applications between clients and servers over WebSockets. It provides Ratchet for serving, Pawl as a client, and RFC6455 for I/O-agnostic protocol handling.
```

Comment:
I think "real time" should be hyphenated, but "bi-directional" can lose the hyphen and be one word.

It's a little confusing that Ratchet is both the set of libraries but also a single component, if I'm reading it right. "Ratchet is a set of libraries..." versus "It provides Ratchet...". I'm not sure how that could be clarified.

Could "It provides Ratchet" be rephrased to "Ratchet is the server library"?